### ISSUE TITLE
trouble with multicols and the latex writer

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#785]
```
#!rst
========= ============ ===================
Target
---------------------- -------------------
...
```

This is the generated latex:

```
#!tex
\begin{tabular}{|p{0.317\linewidth}|p{0.317\linewidth}|p{0.317\linewidth}|}
\hline
 \multicolumn{2}{|l|}{\textbf{
Target
} & \textbf{}\\\hline
...
```

I miss a closing ##}## after the last ##\textbf{}##

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/785
- Originally reported by: Benoît Allard
- Originally created at: 2011-10-11T17:03:12.161

**Comment by shimizukawa** [comment]
_From Benoît Allard on 2011-10-11 15:12:18+00:00_

I got the placement of the missing accolade wrong: it should be between the ##}## on the last line and the ##&##

**Comment by shimizukawa** [comment]
_From Benoît Allard on 2011-10-12 13:28:47+00:00_

The trouble actually is that only one ##context.pop()## is done, where multiple stuff could have been appended to the context.

There does not seems to be any place in this writer where this kind of stuff is taken into account.

A solution could be to append() the actual number of stuff that should be pop()ed, then pop(), if a number, then pop() that number of time ...

**Comment by shimizukawa** [comment]
_From Benoît Allard on 2011-10-12 18:44:23+00:00_

See https://bitbucket.org/birkenfeld/sphinx/pull-request/29/, solved it a better way.

**Comment by shimizukawa** [comment]
_From Benoît Allard on 2011-10-23 21:15:56+00:00_

Fixed in <<cset 09d27b64c64f270cd91d4769e17064fde3974769>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:33:11+00:00_

Removing version: 1.1 (automated comment)
